aiida_quantumespresso/utils/validation/trajectory.py

```diff
@@ -54,7 +53,7 @@ def verify_convergence_forces(trajectory, index=-1, threshold=None):
     try:
         forces = trajectory.get_array('forces')[index]
     except (KeyError, IndexError):
-        return None
+        return ValueError('the `forces` array does not exist or the given index exceeds the length.')
```

Comment:
Should this be a `raise` instead of `return`?

Reply:
Damn straight :)

Reply:
Almost missed it, too 😅 Seems almost like something `pylint` should warn about.